sklearn/utils/tests/test_extmath.py

```diff
@@ -99,38 +99,50 @@ def test_logsumexp():
     assert_array_almost_equal(np.exp(logsumexp(logX, axis=1)), X.sum(axis=1))
 
 
-def test_randomized_svd_low_rank():
+def check_randomized_svd_low_rank(dtype):
```

Comment:
nitpick: I prefer to keep the word `test` instead of `check`.
It is a unit test, not a input check inside the code.